packages/core/stories/combo-box/combo-box.qa.stories.tsx

```diff
@@ -213,3 +214,28 @@ ClosedExamples.parameters = {
     },
   },
 };
+
+export const OpenWithSingleSelectionExamples: StoryFn<
+  QAContainerProps
+> = () => (
+  <QAContainer cols={2} itemPadding={12} width={800}>
+    <StackLayout gap={8}>
+      <ComboBox placeholder="State" open defaultSelected={["Alaska"]}>
+        <Option value={"Alaska"} />
+      </ComboBox>
+      <ComboBox multiselect open defaultSelected={["Alaska"]}>
+        <Option value={"Alaska"} />
+      </ComboBox>
+    </StackLayout>
+    <ComboBox open defaultSelected={["Alaska"]} style={{ marginBottom: 200 }}>
+      <OptionGroup label={"A"}>
+        <Option value={"Alaska"} />
+      </OptionGroup>
+      <OptionGroup label={"C"}></OptionGroup>
```

Comment:
Is this group intentionally empty?

Reply:
yes, it was just to make sure the item border was not going to overlap with the group border

Reply:
and I guess adding a option would make the list longer?

Reply:
We should add a comment, it seems something to easily miss.